pyproject.toml

```diff
@@ -601,12 +601,7 @@ select = [
     "N815", # Variable {name} in class scope should not be mixedCase
     "PERF", # Perflint
     "PGH004", # Use specific rule codes when using noqa
-    "PLC0414", # Useless import alias. Import alias does not rename original package.
-    "PLC", # pylint
-    "PLE", # pylint
-    "PLR", # pylint
-    "PLW", # pylint
-    "Q000", # Double quotes found but single quotes preferred
```

Comment:
This Q was enabled. I know there are changes in python 312 about being able to use double quotes inside f strings. But not sure this does not cover a broader case than that

Reply:
The rule `Q000` is enabled and ignored at the same time. An it is one of the rules which should be disabled when using the ruff formatter, because the formatter handles it. https://docs.astral.sh/ruff/rules/bad-quotes-inline-string/#formatter-compatibility